src/NumericalAlgorithms/LinearOperators/PowerMonitors.hpp

```diff
@@ -45,4 +51,80 @@ std::array<DataVector, Dim> power_monitors(const DataVector& input_data_vector,
                                            const Mesh<Dim>& mesh);
 /// @}
 
+/*!
+ * \ingroup SpectralGroup
+ * \brief Returns the maximum value of a variable in the element.
+ *
+ * Compute the maximum of variable \f$ u \f$ in the element as
+ *
+ * \f{align*}{
+ *  u_\mathrm{max} = \max_{ijk} \left\lvert u_{ijk} \right\rvert ,
+ * \f}
+ *
+ * where \f$ u_{ijk}\f$ are the nodal coefficients
+ * of variable \f$ u \f$.
+ *
+ */
+double maximum_of_variable(const DataVector& input_data_vector);
```

Comment:
Do we really need 20 lines of code to wrap `max(abs(u))`? This function currently only gets used in one place.

Reply:
True. I'll remove it.